packages/blockchain-wallet-v4-frontend/src/scenes/Interest/index.tsx

```diff
@@ -176,7 +176,11 @@ class Interest extends React.PureComponent<Props, StateType> {
               <TransactionList />
             </LazyLoadWrapper>
           ),
-          Failure: () => null,
+          Failure: () => (
+            <Text size='16px' weight={500}>
+              Oops. Something went wrong. Please refresh and try again.
```

Comment:
Should we here use static text or maybe move it into translations file?

Reply:
Also, maybe we could use the error from `e`... `Failure: (e) => `

This helps Platform and support